assets/src/edit-story/app/media/media3p/api/media3pApiProvider.js

```diff
@@ -77,14 +84,20 @@ function Media3pApiProvider({ children }) {
   async function listMedia({
     provider,
     searchTerm,
+    selectedCategoryName,
     orderBy,
     mediaType,
     pageToken,
   }) {
     if (provider.toLowerCase() !== Providers.UNSPLASH) {
       throw new Error(`Unsupported provider: ${provider}`);
     }
-    const filter = constructFilter(provider, searchTerm, mediaType);
+    const filter = constructFilter(
```

Comment:
Should this throw an exception if both searchTerm and selectedCategory are present?

Reply:
Done.